test/server/admin/stats_handler_test.cc

```diff
@@ -232,14 +239,16 @@ TEST_P(AdminStatsTest, HandlerStatsPlainTextHistogramBucketsDisjoint) {
 
   store_->mergeHistograms([]() -> void {});
 
+  setUseRe2Filters(false);
```

Comment:
Should the tests be parameterized on this value, instead of having a mix of values in different tests?

Reply:
Done -- there was a bit of complexity in this change because AdminInstanceTest is already parameterized on ipv6 vs ipv4, and is shared with other tests.

However I worked around it by noting that most of the tests were not actually sensitive to ip version, so I just made each individual test parameterized on what makes sense for it